optuna/dashboard.py

```diff
@@ -18,6 +18,9 @@
     from typing import Optional  # NOQA
 
 with try_import() as _imports:
+    from distutils.version import StrictVersion
```

Comment:
How about importing this at the top, at the top indentation level, to not by mistake catch errors related to `StrictVersion`?

Reply:
Thank you. Fixed in https://github.com/optuna/optuna/pull/1326/commits/92ad990ff420f7428b4792a2a38912aa55d669d2.